controls/V-92963.rb

```diff
@@ -55,16 +55,16 @@
 
   domain_role = command('wmic computersystem get domainrole | Findstr /v DomainRole').stdout.strip
 
-  if domain_role == '4' || domain_role == '5'
+  #if domain_role == '4' || domain_role == '5'
```

Comment:
Not sure why this is commented out. The if statement ensures that the test is skipped if the target is not a domain controller (as indicated in the [check text](https://github.com/mitre/microsoft-windows-server-2019-stig-baseline/blob/master/controls/V-92963.rb#L16))

Reply:
Thanks for uncommenting this condition. Resolving conversation.